src/box/iproto.cc

```diff
@@ -2660,8 +2728,23 @@ tx_process_override(struct cmsg *m)
 		body_end = body + msg->header.body[0].iov_len;
 	}
 
+	/*
+	 * If we took the `override_route', there must exist either request
+	 * type-specific or unknown request type handler. Their availability
+	 * is checked by the IPROTO thread.
+	 */
```

Comment:
Yes. If I'm not wrong, `iproto_thread->req_handlers` and `tx_req_handlers` are always in sync, because `iproto_do_cfg()` is implemented via synchronous call over cbus. I'd also ask @CuriousGeorgiy to verify this.

Reply:
So, if `iproto` -> `tx` messages are asynchronous, there is a chance a message was sent, but the handlers were deleted. Hence, this comment and an assertion below are not fair.

Reply:
Discussed f2f, it's OK.